packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -183,6 +183,34 @@ enum SchedulerPhase {
   postFrameCallbacks,
 }
 
+/// This callback is invoked when a request for [DartPerformanceMode] is disposed.
+///
+/// See also:
+///
+/// * [PerformanceModeRequestHandle] for more information on the lifecycle of the handle.
+typedef _PerformanceModeCleaupCallback = VoidCallback;
```

Comment:
You can just delete this and use `VoidCallback` directly wherever `_PerformanceModeCleaupCallback` is currently used.

Reply:
I think it's useful for the reader, makes it easy to understand that this type is called when the performance mode is resolved.  Let me know if you feel strongly about it, and I can change it up in a follow-up PR.